Shared/Model/Category.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Category.swift
+//  Fun Gen
+//
+//  Created by Julian Burrington on 4/23/22.
+//
+
+import Foundation
+
+enum Category: Codable {
+    case outdoor, game, destination, movie, tvShow, food, drinking, custom
```

Comment:
How do you envision the `custom` category to be used? It sounds like a collection of many possible categories, but the current representation can't capture what the actual category will be.

Reply:
Correct. I think for now we were trying to have our categories cover almost everything (ignoring subcategories) and custom would be very rarely used.

The stretch goal for categories (suggested by our TA) was to have the app suggest top used options by the user in respective categories. So if user always suggests 'Among Us' for activities that are of category 'game', then that would be at the top of suggestions displayed to this user in this stretch feature.

To solve the issue of 'custom' potentially being a collection of numerous categories, we could in this stretch feature label the app's suggestions of top sorted options as 'Recently Used'. So for almost all users the 'Recently Used' would be relevant and match the appropriate category, but for custom activities it may not return an option the user wants to reuse.

What do you think?